nitrite_test.go

```diff
@@ -7,79 +7,110 @@ import (
 	"testing"
 	"time"
 
-	"github.com/blocky/nitrite"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
+	"github.com/blocky/nitrite"
 )
 
-// An easy way to create a new nitro attestation is to use delphi-cli.
-// Request an attestation over some endpoint and copy the resulting
-// "enclave_attestation" string from the output---it should be a
-// base64 encoded string
-//
```

Comment:
I think this was one of the few times where Dave recommended I add a comment. The reason being it can be difficult for devs to determine where complex test data (such as an attestation) came from and how to re-create it should they need to. So, if this is still a valid way for creating test data I would leave the comment

Reply:
OK, great, I updated the comment